setup.py

```diff
@@ -55,6 +55,8 @@
             # work with our custom page template.
             'Sphinx~=1.5.6',
             'sphinx_rtd_theme',
+            # XXX: Temporarily freeze docutils to 0.15.2 since Sphinx has
```

Comment:
The main problem is that sphinx has [too open requirements](https://github.com/sphinx-doc/sphinx/blob/fb9adbf0be0d136cc61ad5cc7b45dae62f133aa9/setup.py#L26) and doesn't work with them. The second problem is that we install pre-release versions of all dependencies instead of just for Resolwe.